packages/xo-web/src/xo-app/host/install-certificate.js

```diff
@@ -0,0 +1,109 @@
+import _ from 'intl'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import React from 'react'
+import SingleLineRow from 'single-line-row'
+import { Col, Container } from 'grid'
+import { form } from 'modal'
+import { generateId } from 'reaclette-utils'
+import { installCertificateOnHost } from 'xo'
+import { provideState, injectState } from 'reaclette'
+import { Textarea as DebounceTextarea } from 'debounce-input-decorator'
+
+const InstallCertificateModal = decorate([
+  provideState({
+    effects: {
+      onChange(_, { target: { name, value } }) {
+        const { props } = this
+        props.onChange({
+          ...props.value,
+          [name]: value,
+        })
+      },
+    },
+    computed: {
+      inputCertificateChainId: generateId,
+      inputCertificateId: generateId,
+      inputPrivateKeyId: generateId,
+    },
+  }),
+  injectState,
+  ({ state, effects, value }) => (
+    <Container>
+      <SingleLineRow>
+        <Col mediumSize={4}>
+          <label htmlFor={state.inputCertificateId}>
+            <strong>{_('certificate')}</strong>
```

Comment:
Don't need `strong` for `label` as for other modals in XO.

Reply:
The modal is structured as key/value, we generally make keys bold. This UI representation is used in other modals like https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/backup/new/new-schedule.js#L77